modules/custom/az_paragraphs/az_paragraphs.info.yml

```diff
@@ -5,3 +5,5 @@ core_version_requirement: ^8.8 || ^9
 package: 'The University of Arizona'
 dependencies:
   - paragraphs
+  - az_card
```

Comment:
field storage for the card field relies on az_card
field storage for text with background relies on az_media